src/pulsar_spectra/catalogue_papers/Murphy_2017_raw_to_yaml.py

```diff
@@ -1,6 +1,13 @@
 import yaml
 
-pulsar_dict = {}
+from pulsar_spectra.scripts.csv_to_yaml import dump_yaml
+
+pulsar_dict = {
+    "Paper Metadata": {
+        "Data Type": "Beamforming",
+        "Observation Span": "Single-epoch",
```

Comment:
Same thing as with Mantovanini_2025, these are measured from mosaics so "Several-epoch" may be more appropriate